src/emulator/eventarcEmulator.ts

```diff
@@ -10,7 +10,7 @@ import { EmulatorRegistry } from "./registry";
 import { FirebaseError } from "../error";
 import { cloudEventFromProtoToJson } from "./eventarcEmulatorUtils";
 
-interface CustomEventTrigger {
+interface EmulatedEventTrigger {
```

Comment:
*nit - how about EventarcTrigger? I don't feel strongly about this but it may make sense to apply a more specific naming convention, in case we add other event trigger interfaces for other emulated services.